e2e-test/src/tests/delete.test.js

```diff
@@ -0,0 +1,65 @@
+import {
+  vBtn,
+  vCard,
+  vIcon,
+  vListTile,
+} from 'jest-puppeteer-vuetify';
+import { CLIENT_URL, deleteAllDatasets, allCleared } from '../util';
+
+
+describe('delete dataset', () => {
+  it('delete one dataset', async () => {
+    await Promise.all([
+      page.goto(CLIENT_URL),
+      page.waitForNavigation({ waitUntil: 'networkidle0' }),
+    ]);
+    await expect(page).toClickXPath(vBtn('Your Datasets'));
+
+    await deleteAllDatasets();
+
+    // todo: make a function for upload file(s)
+    const fileInputElementHandle = await page.waitForXPath("//input[@type='file']");
+    await fileInputElementHandle.uploadFile('../e2e-test/data/deleteTest.csv');
+    await page.waitForXPath(vListTile({ title: 'deleteTest.csv ' }));
+    await page.waitForXPath(vListTile({ title: 'deleteTest.csv ', content: 'Dataset ready for analysis' }));
+
+    await expect(page).toClickXPath(vBtn(vIcon('mdi-close')));
+    await page.waitForXPath(vCard({ headline: 'Really delete 1 dataset?' }));
+    await expect(page).toClickXPath(vBtn('Delete'));
```

Comment:
This code isn't working, and neither is the assert. As a general rule you should always deliberately break your tests once you've written them to make sure that they would correctly fail when something breaks.
This code is running too fast; I think it's deleting the item before it's been saved to the database, so the item is still there with a little `FATAL ERROR` chip attached to it. Adding a `await new Promise(resolve => setTimeout(resolve, 1000));` will probably fix it.